src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/reducers/__tests__/chartBuilderReducer.test.ts

```diff
@@ -966,5 +1152,371 @@ describe('chartBuilderReducer', () => {
         },
       });
     });
+
+    test('has correct state with initial configuration for a map', () => {
+      const initialConfiguration: Chart = {
+        legend: {
+          position: 'top',
+          items: [],
+        },
+        axes: {
+          major: {
+            type: 'major',
+            dataSets: [
+              {
+                indicator: 'indicator-1',
+                filters: ['filter-1'],
+              },
+            ],
+            referenceLines: [],
+          },
+        },
+        type: 'map',
+        height: 300,
+        title: 'Chart title',
+        alt: '',
+        boundaryLevel: 1,
+      };
+
+      const { result } = renderHook(() =>
+        useChartBuilderReducer({
+          data: testFullTable.results,
+          meta: testFullTable.subjectMeta,
+          tableTitle: 'Table title',
+          chart: initialConfiguration,
+        }),
+      );
+
+      expect(result.current.state).toEqual<ChartBuilderState>({
+        axes: {
+          major: {
+            type: 'major',
+            groupBy: 'locations',
+            dataSets: [
+              {
+                indicator: 'indicator-1',
+                filters: ['filter-1'],
+              },
+            ],
+            referenceLines: [],
+            visible: true,
+            label: {
+              text: '',
+            },
+          },
+        },
+        definition: mapBlockDefinition,
+        map: { dataSetConfigs: [] },
```

Comment:
Left a previous comment that we're not testing anything of value currently because we're not actually calculating any of the map's initial `dataSetConfigs`. Seems like that's still the case.

Seems like the problem might be because the data we're supplying doesn't actually match up to the data sets we've configured in `axes.major`.